lib/src/server/sse_handler.dart

```diff
@@ -221,9 +255,10 @@ class SseHandler {
       shelf.Request req, String path) async {
     try {
       var clientId = req.url.queryParameters['sseClientId'];
+      var messageId = int.parse(req.url.queryParameters['messageId'] ?? '0');
```

Comment:
do we have to have this '0'? It does seem reasonable to make this a breaking change and require the server to pass a `messageId`.

Reply:
No. We explicitly want this to be backwards compatible. We can't really control when the Dart Debug Extension will be updated so there will be a period of time where the client does not send the message ID.